opentelemetry-semantic-conventions/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## vNext
 
+## v0.26.0
+### Changed
+
+- Starting with this version, this crate will use Weaver for the generation of
+  the semantic conventions.
+- Introduced a new feature `semconv_experimental` to enable experimental semantic conventions.
```

Comment:
do you anticipate further features flags per signal/area so users can just opt-in to those that they need? 
Are there concerns usually about too big assembly with all the unused constants?

Reply:
We could indeed introduce a feature for each namespace (e.g., `db`, `container`, `url`, etc.) and for each signal type (e.g., `metric`, `span`, `event`, etc.). This would allow us to use configurations like `#[cfg(any(feature = "metrics", feature = "db"))]`. 

This should be simple to implement, so please let me know if you’d like to proceed in this direction.

Reply:
not for this PR, I wanted to ask if it is feasible/easy, if demand arises for such a need!